content/en/aws/opensearch/index.md

```diff
@@ -0,0 +1,415 @@
+---
+title: "OpenSearch Service"
+linkTitle: "OpenSearch Service"
+categories: ["LocalStack Community"]
+description: >
+  Amazon OpenSearch Service
```

Comment:
Maybe we should add the "(successor to Amazon Elasticsearch Service)" which AWS uses in its description: https://aws.amazon.com/opensearch-service

Reply:
Good idea, I added the note.